### PR TITLE
Transferring non-album Google photos

### PR DESCRIPTION
**Pull request by olsona**
Transferring non-album Google photos - next step is to alter all of the Importers to deal with duplicates correctly.
Making Exporter and Importer throw errors for better retrying.

### CODE REVIEW

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Exporter.java [hunk 1]
```diff
@@ -22,17 +22,22 @@
...
 
   /**
    * Performs an export operation, starting from the data specified by the continuation.
-   *  @param jobId the job id
+   *
+   * @param jobId the job id
    * @param authData authentication data for the operation
    * @param exportInformation info about what data to export see {@link ExportInformation} for more
    */
   // REVIEW: The original throws IOException. Continue to use checked
```
Comment (rtannenbaum) [inline portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Exporter.java]: Can the two review's be consolidated?

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java [hunk 2]
```diff
@@ -31,5 +31,5 @@
    * @return the operation result
    */
   // REVIEW: The original throws IOException. Continue to use or return as part of the result?
```
Comment (rtannenbaum) [inline portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java]: review note out of date?

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Exporter.java (hunk 1 above)
Comment (olsona) [reply portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Exporter.java]: Clearer comments.

File: portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java (hunk 2 above)
Comment (olsona) [reply portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java]: Clearer comments.

**Pull request merged**